packages/analytics-core/package.json

```diff
@@ -29,12 +29,12 @@
     "publish": "yarn npm publish --tolerate-republish --access public"
   },
   "dependencies": {
-    "yup": "^0.32.9"
+    "yup": "^0.32.11"
```

Comment:
Can we use yup 1.x?

Reply:
I think we should do it in its own ticket. I've had too many headaches with `yup` in the past